### PR TITLE
Add libcmaes

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge/core it seems on travis, build dependencies are not installed for the next build (here autoconf).

**Comment by moorepants** [comment]
I'm not sure if there is a convention yet but, for example, for the symengine library it only installs the C++ code if you install "symengine" and if you install "python-symengine" it will install the former plus the python bindings. It may be worth separating them for those that only want to use the C++ lib. It also seems like packages in conda are named without the prefix "lib" more often than not.

**Comment by jschueller** [comment]
- I don't see the point for a package manager that is python-oriented ; the point is having packages for python, not a linux distro. However the discussion exists right now for boost in packages as it's widely used and that can cause dependency problems, but frankly for libcmaes I think this is overkill.
- I see plenty of recipes that begin with lib*, just type "lib" in the conda-forge organisation repo search bar

**Comment by jschueller** [comment]
@conda-forge/core ping

**Comment by scopatz** [comment]
Please enable CircleCI on your fork: http://conda-forge.github.io/docs/ciservices.html#circleci-linux

File: recipes/libcmaes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+requirements:
+  build:
+    - toolchain
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - python
+    - boost 1.62.*
+    - eigen
+    - numpy
```
Comment (scopatz) [inline recipes/libcmaes/meta.yaml]: Please pin numpy to `numpy x.x`

**Comment by jschueller** [comment]
@scopatz, done. but the 2nd build fails on circleci as autotools are not found, is this a known bug ?

**Comment by scopatz** [comment]
That is a new one to me.  Have you merged from matser recently? There have been some conda-build updates lately

**Comment by jschueller** [comment]
@scopatz you were right, rebasing made it pass

**Comment by moorepants** [comment]
Here is a thread on naming conventions with some relevant info: https://groups.google.com/forum/#!topic/conda-forge/VsEPbfNA0GE. Doesn't seem to be a strong convention yet though.

**Comment by scopatz** [comment]
@moorepants yeah I am not sure there will be. 

@jschueller Great! If you are happy with it, let me know and I'll merge it.

**Comment by jschueller** [comment]
@scopatz this is ready to go

**Pull request merged**